keyman_trust_darwin.go

```diff
@@ -20,7 +20,21 @@ func DeleteTrustedRootByName(commonName string, prompt string) error {
 	return nil
 }
 
-func (cert *Certificate) AddAsTrustedRoot(prompt string) error {
+// IsInstalled checks whether this certificate is install based purely on looking for a cert
+// in the system keychain that has the same common name.  This function returns
+// true if there are one or more certs in the system keychain whose common name
+// matches this cert.
+func IsInstalled(cert *Certificate) bool {
```

Comment:
I see that you removed this completely for Linux (which makes sense). So should we just make it private now, since it's not supported on all platforms?

Also, I find `cert.IsInstalled` more fluent than `IsInstalled(cert)`. Is there a reason you changed that?

Reply:
@oxtoacart - no specific reason. Not sure what I was thinking :)
I'll turn it private